newsletter-january-2023.rst

```diff
@@ -0,0 +1,83 @@
+.. post:: Jan 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - January 2023
+=======================================
+
+Happy 2023! In this newsletter, we present the updates happening in 2022's last month and since the :doc:`previous newsletter </newsletter-december-2022>`.
+
+.. Commenting out this stuff, isn't it better that 
+.. 
+.. News and updates
+.. ----------------
+.. 
+.. Here are the latest updates from our team:
+
+- 📹️ Eric's talk at DjangoCon US can be watched here: `Documenting Django Code in 2022`_
+- 🛠️ We are busy reorganizing our own documentation. Since last newsletter, we have opened or merged 20 documentation PRs. We look forward to a "show and tell" about how all of this was refactored using the `Diátaxis Framework`_.
+- 🚢️ Sphinx 6.0 and 6.1 have been released. Read :doc:`our considerations for upgrading </sphinx6-upgrade>`.
+- ⚙️ Add new stuff here
+- 💡️ Choose your own emoji
+- 🔍️ 404 pages are being improved by contextualization the user message, giving relevant guidance to readers and project owners.
```

Comment:
Didn't we already talked about this in a previous newsletter?

Reply:
It seems we have the same line in the December newsletter: https://blog.readthedocs.com/newsletter-december-2022/

BTW, I don't like this patter of copy&paste the previous one without removing the old content. Iit's hard to know what's really going out in _this one in particular_. If it's a placeholder, it should be clear it's a placeholder. Otherwise, it's confusing for reviewers.

Reply:
Sure, like this is not open "for review" but a draft where you can add ideas.